tests/test_decam.py

```diff
@@ -40,6 +40,7 @@ def test_decam_translator(self):
                     exposure_id=229388,
                     exposure_group="229388",
                     exposure_time=200.0 * u.s,
+                    focus_z=2497.32 * u.um,
```

Comment:
There should be a test in the megaprime test file to ensure it's 0.0mm.

Reply:
Added test to `tests/test_cfht.py`